lib/pdk/Header.v3

```diff
@@ -0,0 +1,49 @@
+// `Header` represents an HTTP Request header.
+// Multiple values for a single key are not deduped.
+class Header {
+  var m: HashMap<string, string>;
```

Comment:
If `m` is immutable, you could do the more "Virgilic" way of having `def m = Strings.newMap<string>();` right in the field declaration. It's generally good to have at least full-word field names, so `map` might be a better name.